artifactory.py

```diff
@@ -1502,6 +1506,10 @@ class PureArtifactoryPath(pathlib.PurePath):
 
     parser = _artifactory_flavour
     _flavour = parser  # Compatibility shim for Python < 3.13
+    if IS_PYTHON_3_13_OR_NEWER:
+        import glob
+
+        _globber = glob._Globber
```

Comment:
where is this used ?

Reply:
can you please add the comment here then ?

Reply:
Yes, I've added a comment, but I also ended up figuring out the remaining issue, which required me to change this to a custom subclass of _Globber, so I've changed line a bit more significantly in https://github.com/devopshq/artifactory/pull/476/commits/1bcbd5910f1508fff2439b417f7c1dbb27dc858b